### DISCUSSION TITLE
xpath绝对路径执行问题

### DISCUSSION THREAD

**Discussion by zhougy1989** [SonicCloudOrg/sonic-server#466] [Q&A]
在远程控制里执行自动化脚本，绝对路径是可以执行通过的
![image](https://github.com/user-attachments/assets/51fd57f6-da06-4064-9b3b-dcd97cc87d4f)

但是在定时任务里或者测试套件里执行，绝对路径就找不到，是什么情况
![image](https://github.com/user-attachments/assets/cb619c3a-f49a-4d38-9941-d415ccbfb951)

**Comment by ZhouYixun** [comment]
同一个设备吗
